scheduler/src/cook/mesos/api.clj

```diff
@@ -135,8 +279,55 @@
 (defn validate-and-munge-job
   "Takes the user and the parsed json from the job and returns proper
    Job objects, or else throws an exception"
-  [db user task-constraints {:strs [cpus mem uuid command priority max_retries max_runtime name uris ports env] :as job}]
-  (let [munged (merge
+  [db user task-constraints {:strs [cpus mem uuid command priority max_retries max_runtime name uris ports env container] :as job}]
+  (let [make-docker
```

Comment:
Let's try if http://stackoverflow.com/questions/9406156/clojure-convert-hash-maps-key-strings-to-keywords can achieve the same thign here